modules/travel_pay/app/services/travel_pay/token_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class TokenService
+    #
+    # GET TOKENS
+    # returns a hash containing the veis_token & btsss_token
+    ##
+    #
+    def get_tokens(current_user)
+      veis_token = token_client.request_veis_token
+      btsss_token = token_client.request_btsss_token(veis_token, current_user)
+
+      { 'veis_token' => veis_token,
+        'btsss_token' => btsss_token }
+    end
+
+    private
+
+    ##
+    # Syntactic sugar for determining if the client should use
+    # fake api responses or actually connect to the BTSSS API
+    def mock_enabled?
+      Settings.travel_pay.mock
+    end
```

Comment:
I don't think this method is used in this class.

Reply:
Yup, nice catch! Thanks!